docs/deployment_practices.md

```diff
@@ -74,7 +74,7 @@ In your SSL (port 443) virtual host, set up HSTS and use these settings to give
     SSLProtocol all -SSLv2 -SSLv3
     SSLHonorCipherOrder on
     SSLCompression off
-    SSLCipherSuite EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:EECDH+RC4:RSA+RC4:!MD5
```

Comment:
I'm fine with this change as long as it doesn't make landing pages inaccessible for a substantial subset of users (due to browser compatibility issues).